otx/algorithms/action/adapters/mmaction/models/detectors/fast_rcnn.py

```diff
@@ -69,6 +75,8 @@ def __init__(self, backbone, roi_head, train_cfg, test_cfg, neck=None, pretraine
             pretrained=pretrained,
         )
         self.detector = None
+        self.deply_cfg = None
```

Comment:
> Change building method 2d person detector: mmdeploy requires to build model using its processor so that the monkey patched function works properly. Therefore method to add 2d person detector is changed

There is nothing special in `init_pytorch_model` method. This should be reverted.